deploy/coins/monero/scripts/entry-dev.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# Setup Environment
+RPC_USER="${RPC_USER:=serai}"
+RPC_PASS="${RPC_PASS:=seraidex}"
+MINER="${MINER:=xmraddr}"
+BLOCK_TIME=${BLOCK_TIME:=5}
+
+
+# Run Monero
+monerod --regtest --rpc-login ${RPC_USER}:${RPC_PASS} \
```

Comment:
`./monerod --regtest --offline --fixed-difficulty=1 --detach`

is sufficient if the tests control block production. If we want tests which have a block every few seconds, I'd rather the test spawn an async test to do that on its end. That way it can still run tests where it does control block timing.

Also, your miner command will spam block addition as fast as it can process. NACK for that.

There's also no need for RPC access controls, except perhaps the foreign IP ones, UNLESS the foreign IP makes Monero mandates a login. My preferred method would then be to override the login, unless its an external (as in, external to Docker) access control measure. Then my question is on the security of a static login and why this is externally accessible.